oak_functions/loader/src/server.rs

```diff
@@ -411,8 +434,8 @@ impl wasmi::Externals for WasmState {
                     .take()
                     .expect("no extensions_indices is set");
                 let extension = match extensions_indices.get_mut(&index) {
-                    Some(extension) => Box::new(extension),
-                    None => panic!("Unimplemented function at {}", index),
+                    Some(Extension::Native(extension)) => Box::new(extension),
+                    _ => panic!("Unimplemented function at {}", index),
```

Comment:
I'd suggest distinguishing between `None` and `Some(Extension::Uwabi(...))`, perhaps with a different panic message for UWABI. It is a good practice to keep the match arms exhaustive.

Reply:
Done.